Chapter08/recipe-06/cxx-example/src/evolution/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+target_sources(
+  automaton
+  PRIVATE
+    ${CMAKE_CURRENT_LIST_DIR}/evolution.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/evolution.hpp
+  )
+
+target_include_directories(
```

Comment:
Ditto. Maybe this is not needed if you mark the header as a `PUBLIC` source file